requirements.txt

```diff
@@ -1,15 +1,53 @@
-colorlover==0.2.1
-dicttoxml==1.6.6
-latexcodec==1.0.3
-monty==0.8.1
-plotly==1.12.0
-pybtex==0.20.1
-pymongo==3.2.2
-pyspin==1.0.2
-python-dateutil==2.5.3
-pytz==2016.4
-PyYAML==3.11
-requests==2.9.1
-six==1.10.0
-tqdm==4.7.4
-xmltodict==0.10.1
+aioitertools==0.7.0
+attrs==19.3.0
+bcrypt==3.1.7
+certifi==2020.4.5.1
+cffi==1.14.0
+chardet==3.0.4
+click==7.1.2
+colorlover==0.3.0
+cryptography==2.9.2
+dicttoxml==1.7.4
+dnspython==1.16.0
+fastapi==0.54.2
+Flask==1.1.2
+futures==3.1.1
+idna==2.9
+importlib-metadata==1.6.0
+itsdangerous==1.1.0
+Jinja2==3.0.0a1
+jsonschema==3.2.0
+latexcodec==2.0.0
+MarkupSafe==2.0.0a1
+mongogrant==0.3.1
+mongomock==3.19.0
+monty==3.0.2
+msgpack==1.0.0
+numpy==1.19.0rc1
+paramiko==2.7.1
+plotly==4.7.1
+pybtex==0.22.2
+pycparser==2.20
+pydantic==1.5.1
+pydash==4.7.6
+pymongo==3.10.1
+PyNaCl==1.3.0
+pynng==0.5.0
+pyrsistent==0.16.0
+pyspin==1.1.1
+python-dateutil==2.8.1
+pytz==2020.1
+PyYAML==5.3.1
+requests==2.23.0
+retrying==1.3.3
+sentinels==1.0.0
+six==1.14.0
+sniffio==1.1.0
+sshtunnel==0.1.5
+starlette==0.13.2
+tqdm==4.46.0
+typing-extensions==3.7.4.2
+urllib3==1.25.9
+Werkzeug==1.0.1
+xmltodict==0.12.0
+zipp==3.1.0
```

Comment:
Something is wrong here. You don't use most of these in the code.

Reply:
shortened it, i did a `pip freeze > requirements.txt`, it seems like it also spitted out the requirements from installing `maggma` and other packages

Reply:
try `pip-chill`: https://pypi.org/project/pip-chill/